exercises/isogram/src/example/java/Isogram.java

```diff
@@ -0,0 +1,29 @@
+package exemple;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Isogram {
+	
+	public static boolean isIsogram(String word){
+		
+		Set<Character> charset= new HashSet<Character>();
+		String[] words = word.split(" ");
+		String newword = concat(words);
+		words = newword.split("-");
+		newword = concat(words).toLowerCase();
+		
+		for(int i = 0; i < newword.length(); i++)
+			charset.add(newword.charAt(i));
```

Comment:
Usually we like to be explicit with our curly braces, even in short blocks like this.